robot/EDA/resources/EDA.py

```diff
@@ -559,3 +559,32 @@ def verify_app_exists(self,app):
         """Verifies that the given app is present in the app launcher"""
         locator=eda_lex_locators["eda_settings"]["app_tile"].format(app)
         self.selenium.wait_until_page_contains_element(locator,timeout=60,error=f'{app} did not open in 1 min')
+
+    @capture_screenshot_on_error 
+    def checkbox_status(self,cbx_name,status):
+        """verifies if the specified checkbox is with expected status in readonly mode"""
+        locator=locator(cbx_name,status)
+        self.selenium.page_should_contain_element(locator)
+
+    def choose_frame(self, value):
+        """Returns the first displayed iframe on the page with the given name or title"""
+        locator = eda_lex_locators['frame'].format(value,value,value)
+        frames = self.selenium.get_webelements(locator)
+        self.selenium.capture_page_screenshot()
+        print(f'list of frames {frames}')
```

Comment:
This appears to be a debug statement. Was it left in intentionally? You might want to change this to a log statement with a log level of debug:

    self.builtin.log(f'list of frames {frames}', "DEBUG")

In scanning through the code I see several `print` statements. I won't repeat myself, but the same comment applies to them all.

Reply:
fixed